packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -30,6 +30,14 @@ class CreateCommand extends Command {
       defaultsTo: true,
       help: 'Whether to run "pub get" after the project has been created.'
     );
+    argParser.addFlag(
+      'with-driver-test',
```

Comment:
Should we have a similar option to create unit tests?  Should we have the ability to create a driver and/or unit test for an existing project?